examples/WebView/ReactNativeWebView.php

```diff
@@ -14,11 +14,11 @@ class ReactNativeWebView extends \LivingMarkup\Module
 {
     public function onRender()
     {
-        if($this->args['initiate'] == "true" && !empty($this->args['object'])) {
+        if ($this->args['initiate'] == "true" && !empty($this->args['object'])) {
             $object = $this->args['object']; // escape JSON
-            return "<script>
+            return "<body><script>
                 window.ReactNativeWebView.postMesage({$object}, '*');
```

Comment:
An arg type can be specified which may help with your example, it was in the spec and recently added. e.g. `<arg name="object" type="json"></arg>`